README.md

```diff
@@ -378,6 +388,40 @@ Note, that the parameter must follow the format "uid:gid" where `uid` and `gid`
 
 Allows specifying the Python 3 interpreter to be used when calling the Pieman-specific utilities. 
 
+#### Extra
+
+##### CREATE_ONLY_ARTIFACT=false
+
+Makes Pieman restrict itself to only creating an artifact (a file with the `.mender` extension) which can later be uploaded to [hosted.mender.io](https://hosted.mender.io) to provide for OTA updates.
+
+Note, that the parameter conflicts with `CREATE_ONLY_CHROOT` and `ENABLE_MENDER`.
+
+##### ENABLE_MENDER=false
+
+Specifies whether to install the Mender client to provide for OTA updates.
+
+Note that the OTA updates support is currently limited to 32-bit Raspbian and Ubuntu for Raspberry Pi 3 Model B.
+
+##### MENDER_ARTIFACT_NAME="release-1_1.6.0"
+
+...
+
+##### MENDER_DATA_SIZE=128
```

Comment:
Would be great if we create a parameter which will extend the last partition if set to true and don't extend if set to to false.
But for now let's add a note that mender data partition will be extended on the first boot.

Reply:
I created a corresponding issue https://github.com/tolstoyevsky/pieman/issues/124.